.gitignore

```diff
@@ -0,0 +1,3 @@
+gulpfile.js
+package.json
```

Comment:
Gulpfile should not be ignored... if you create new directives there you want other contributors to be able to use them as well.

Package.json should not be ignored. If you add a new dependency you want to have it available if you install the dependencies... if this file does not make it to git, the moment you run npm install elsewhere (or after an npm prune) you will be missing dependencies.